src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working.");
```

Comment:
Similarly, the message 'Bulldozer stopped working.' could be more informative. Consider specifying what has concluded, such as 'Bulldozer has finished its earthmoving work.'